frontend/apps/picasso/pages/crowdloan-rewards/claim.tsx

```diff
@@ -0,0 +1,377 @@
+    selectedEthereumAddress: account,
+  });
+
+  const breadcrumbs = [
+    <Link key="Overview" underline="none" color="primary" href="/">
+      Overview
+    </Link>,
+    <Link
+      key="Crowdloan"
+      underline="none"
+      color="primary"
+      href="/crowdloan-rewards"
+    >
+      Crowdloan Rewards
+    </Link>,
+    <Typography key="claims" color="text.secondary">
+      {flow}
+    </Typography>,
+  ];
+
+  const isStable = isEthAccountEligible;
+
+  const standardPageSize = {
+    xs: 12,
+  };
+
+  return (
+    <DefaultLayout breadcrumbs={breadcrumbs}>
+      <Grid
+        container
+        sx={{ mx: "auto" }}
+        maxWidth={1032}
+        rowSpacing={9}
+        columns={10}
+        direction="column"
+        justifyContent="center"
+        pb={9}
+      >
+        <Grid item {...standardPageSize} mt={theme.spacing(9)}>
+          <PageTitle
+            title={isStable ? "Stablecoin Contribution." : "KSM Contribution"}
+            textAlign="center"
+            subtitle="You will be able to check on your positions here."
+          />
+        </Grid>
+        {hasNothingToClaim() && (
+          <Grid item {...standardPageSize} mt={theme.spacing(9)}>
+            <NoEligibleWalletFeaturedBox
+              title={ineligibleText.title}
+              textBelow={ineligibleText.textBelow}
+            />
+          </Grid>
+        )}
+        {!hasNothingToClaim() && (
+          <Grid item {...standardPageSize} mt={theme.spacing(9)}>
+            {isStable ? (
+              <StablecoinClaimForm
+                isClaiming={isPendingAssociate || isPendingClaim}
+                SS58Address={
+                  ethAssociatedOrSelectedAccount
+                    ? `${ethAssociatedOrSelectedAccount.address} (${ethAssociatedOrSelectedAccount.name})`
+                    : "-"
+                }
+                disabled={
+                  !hasStarted ||
+                  availableToClaim.eq(0) ||
+                  nextStep === CrowdloanStep.None
+                }
+                claimedRewards={claimedRewards}
+                amountContributed={contributedAmount}
+                availableToClaim={availableToClaim}
+                totalRewards={totalRewards}
+                readonlyCrowdLoanContribution={true}
+                readonlyAvailableToClaim
+                readonlyTotalPicaVested
+                picassoAccountName={
+                  selectedAccount ? selectedAccount.name : "-"
+                }
+                readonlySS8Address
+                onClaim={async () => {
+                  nextStep === CrowdloanStep.AssociateEth
+                    ? signEthereum().then(useAssociate)
+                    : nextStep === CrowdloanStep.AssociateKsm
+                    ? signPolkadotJs().then(useAssociate)
+                    : nextStep === CrowdloanStep.Claim
+                    ? claim().catch(console.error)
+                    : undefined;
+                }}
```

Comment:
Poor man's match expression to avoid nested ternaries.

```suggestion
  onClaim={() => {
    const operation = {
      [CrowdloanStep.AssociateEth]: () => signEthereum().then(useAssociate),
      [CrowdloanStep.AssociateKsm]: () => signPolkadotJs().then(useAssociate,
      [CrowdloanStep.Claim]: () => claim().catch(console.error)
    }[nextStep];
    
    return operation();
  }}
```